CHANGELOG.md

```diff
@@ -9,6 +9,25 @@ Breaking changes:
 - [core] renamed preference `list.openMode` to `workbench.list.openMode` [#6481](https://github.com/eclipse-theia/theia/pull/6481)
 - [task] changed `TaskSchemaUpdater.update()` from asynchronous to synchronous [#6483](https://github.com/eclipse-theia/theia/pull/6483)
 - [monaco] monaco prefix has been removed from commands [#5590](https://github.com/eclipse-theia/theia/pull/5590)
+- [plugin] webviews are reimplemented to align with [VS Code browser implementation](https://blog.mattbierner.com/vscode-webview-web-learnings/) [#6465](https://github.com/eclipse-theia/theia/pull/6465)
```

Comment:
@benoitf @olexii4 @evidolob @mmorhun I've tried to capture breaking changes required to align with VS Code here. I hope it would be helpful to adopt them in Che. If some APIs don't have enough information please do follow-up PRs during migration.

Reply:
@amiramw @idoprz I'm not sure how it is important for you. Just pinging, since you run plugins in sidecar containers as well.